WSC.js

```diff
@@ -815,7 +820,7 @@ _.extend(DirectoryEntryHandler.prototype, {
                                     var contents = dataa.split('\n')
                                     var validFile = false
                                     for (var i=0; i<contents.length; i++) {
-                                        contents[i] = contents[i].replaceAll('\t', '').replaceAll('\n', '').replaceAll('\r', '')
+                                        contents[i] = contents[i].replaceAll('\t', '').replaceAll('\n', '').replaceAll('\r', '').replaceAll('var', '').replaceAll('const', '').replaceAll('let', '').replaceAll(' ', '')
```

Comment:
```javascript
contents[i] = contents[i].replaceAll('\t', '').replaceAll('\n', '').replaceAll('\r', '').replaceAll('var', '').replaceAll('const', '').replaceAll('let', '').replaceAll(' ', '')
```

What's this for?

Reply:
It replaces all possible ways the user would declare the post/serverjs key so that it can see if the keys are the same

Reply:
I just decided to remove `var` `const` and `let` checkers because the user could use that text in the key